lib/types/json-schema.ts

```diff
@@ -102,22 +108,9 @@ export type JSONSchemaType<T, _partial extends boolean = false> = (
         ? JSONType<"boolean", _partial>
         : never)[]
     } & (T extends number
-      ? {
-          minimum?: number
-          maximum?: number
-          exclusiveMinimum?: number
-          exclusiveMaximum?: number
-          multipleOf?: number
-          format?: string
-        }
+      ? NumberKeywords
       : T extends string
-      ? {
-          type: JSONType<"string", _partial>
```

Comment:
Btw is this intentional that type: is removed? I was wondering before why it was needed...

Reply:
lol, yeah, it was copy pasta, but redundant, so it wasn't caught. One of the benefits of going ahead with pulling the keywords out.